Index: java_InterfaceArayuz/com/aliyagiztar/Belediyeistekleri.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/java_InterfaceArayuz/com/aliyagiztar/Belediyeistekleri.java b/java_InterfaceArayuz/com/aliyagiztar/Belediyeistekleri.java
new file mode 100644
--- /dev/null	(date 1701010608641)
+++ b/java_InterfaceArayuz/com/aliyagiztar/Belediyeistekleri.java	(date 1701010608641)
@@ -0,0 +1,4 @@
+package java_InterfaceArayuz.com.aliyagiztar;
+
+public interface Belediyeistekleri {
+}
Index: java_InterfaceArayuz/com/aliyagiztar/IKanun.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/java_InterfaceArayuz/com/aliyagiztar/IKanun.java b/java_InterfaceArayuz/com/aliyagiztar/IKanun.java
new file mode 100644
--- /dev/null	(date 1701010594982)
+++ b/java_InterfaceArayuz/com/aliyagiztar/IKanun.java	(date 1701010594982)
@@ -0,0 +1,4 @@
+package java_InterfaceArayuz.com.aliyagiztar;
+
+public interface IKanun {
+}
Index: java_InterfaceArayuz/com/aliyagiztar/Arac.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package java_InterfaceArayuz.com.aliyagiztar;\n\npublic class Arac {\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/java_InterfaceArayuz/com/aliyagiztar/Arac.java b/java_InterfaceArayuz/com/aliyagiztar/Arac.java
--- a/java_InterfaceArayuz/com/aliyagiztar/Arac.java	(revision 446c16846166acd53e562dc99264bed1e8ed5e7f)
+++ b/java_InterfaceArayuz/com/aliyagiztar/Arac.java	(date 1701009054171)
@@ -1,4 +1,9 @@
 package java_InterfaceArayuz.com.aliyagiztar;
 
 public class Arac {
+    public static void main(String[] args) {
+
+       private  String plakaNumarası;
+
+    }
 }
Index: java_InterfaceArayuz/com/aliyagiztar/Taksi.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/java_InterfaceArayuz/com/aliyagiztar/Taksi.java b/java_InterfaceArayuz/com/aliyagiztar/Taksi.java
new file mode 100644
--- /dev/null	(date 1701008957390)
+++ b/java_InterfaceArayuz/com/aliyagiztar/Taksi.java	(date 1701008957390)
@@ -0,0 +1,4 @@
+package java_InterfaceArayuz.com.aliyagiztar;
+
+public class Taksi {
+}
Index: java_InterfaceArayuz/com/aliyagiztar/Kamyon.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/java_InterfaceArayuz/com/aliyagiztar/Kamyon.java b/java_InterfaceArayuz/com/aliyagiztar/Kamyon.java
new file mode 100644
--- /dev/null	(date 1701008889583)
+++ b/java_InterfaceArayuz/com/aliyagiztar/Kamyon.java	(date 1701008889583)
@@ -0,0 +1,4 @@
+package java_InterfaceArayuz.com.aliyagiztar;
+
+public class Kamyon {
+}
Index: java_InterfaceArayuz/com/aliyagiztar/Otobus.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/java_InterfaceArayuz/com/aliyagiztar/Otobus.java b/java_InterfaceArayuz/com/aliyagiztar/Otobus.java
new file mode 100644
--- /dev/null	(date 1701009943605)
+++ b/java_InterfaceArayuz/com/aliyagiztar/Otobus.java	(date 1701009943605)
@@ -0,0 +1,14 @@
+package java_InterfaceArayuz.com.aliyagiztar;
+
+public class Otobus extends Arac {
+    public Otobus() {
+    }
+
+    public static void main(String[] args) {
+        private String plakaNumarasi;
+        private short modelYili;
+        private String modelseri;
+
+        private boolean sunucuDurumu;
+    }
+}
Index: java_InterfaceArayuz/com/aliyagiztar/Motor.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/java_InterfaceArayuz/com/aliyagiztar/Motor.java b/java_InterfaceArayuz/com/aliyagiztar/Motor.java
new file mode 100644
--- /dev/null	(date 1701008957391)
+++ b/java_InterfaceArayuz/com/aliyagiztar/Motor.java	(date 1701008957391)
@@ -0,0 +1,4 @@
+package java_InterfaceArayuz.com.aliyagiztar;
+
+public class Motor {
+}
